README.md

```diff
@@ -6,20 +6,26 @@ A CC7 machine or a mac.
 
 ## Setup
 
-We use alibuild, see complete instructions [here](https://alice-doc.github.io/alice-analysis-tutorial/building/) and install O2 (no need to `init` it, just `build` it).
+1. Install O2
+  * We use alibuild, see complete instructions [here](https://alice-doc.github.io/alice-analysis-tutorial/building/) and install O2 (no need to `init` it, just `build` it).
+  
+2. Install the MySQL/MariaDB development package 
+  * CC7 : sudo yum install mariadb-devel
```

Comment:
Actually I needed to install `mariadb-server` in order to get this working. Installing `devel` doesn't help (checked).